Course_Part_04/src/Assignment16_IsPrime.java

```diff
@@ -0,0 +1,25 @@
+public class Assignment16_IsPrime {
+
+    public static void main(String[] args) {
+
+        for (int num = 2; num < 101; num++) {
+            if (isPrime(num)) {
+                System.out.printf("The number %d is prime%n", num);
+            } else {
+                System.out.printf("The number %d is not a prime%n", num);
+            }
+        }
+
+    }
+
+    private static boolean isPrime(int num) {
```

Comment:
per definitie is 1 geen prime, hoe kun je de methode aanpassen dat ie hier ook rekening mee houdt?